### DISCUSSION TITLE
Design guidelines

### DISCUSSION THREAD

**Discussion by vaufatal28** [Remix-Design/RemixIcon#290] [General]
Hi guys,
I really love your icon library! Great job! 
Do you have design guidelines that you apply to all your icons?
Why do I ask it? We want to use your icons in our application and would like to build some icons on our own. Very special not commonly used icons. With a design guideline it would be easier to fit the design of some customer icons to the existing ones of your library. 
If yes, can you share them with the community? 
Thanks in advance and keep up the great work :)
Best wishes, Valerie

**Comment by 42sol-eu** [comment]
I think a design guideline should be not to hard to set up - I am not active in the project, just saying ...

Let us start:

1. only black and white allowed, and background transparent 
2. core format SVG
3. size 32x32
4. svg standard xmlns="http://www.w3.org/2000/svg"
5. view box viewBox="0 0 24 24"
6. xml style - every tag and closing tag on seperate line
7. provide a boxed version too
8. ... more to follow
41. build process ???
42. keep in mind of the minimalistic design (...🤭)

**Comment by xiaochunjimmy** [comment]
We've back and are working on it!
The guideline may published with the next version.

**Reply by vaufatal28** [reply]
Welcome back then :) 
Looking forward to it :)
